http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4110,6 +4154,18 @@ components:
           type: boolean
           description: Whether to turn on autodelete
 
+    UpdateAppConfigRequest:
```

Comment:
Should the valid combination be explained here as well since this is the user facing documentation? Aka users won't go into the leo code to look for it

Reply:
updated